### DISCUSSION TITLE
Is hosting web components in cdnjs possible?

### DISCUSSION THREAD

**Discussion by ghost** [cdnjs/cdnjs#14162] [General] [answered]
Hi all. 

I would like to know if I can submit a pull request(web components) here at cdn.js. The reason for this request would be to make web components made in vanilla javascript more popular and usable with cdn-js.

I would like to do something like this:
```html
<!doctype html>
<html>
<head>
    <meta charset="utf-8">
    <title>Cdn-js for web components</title>
    <!-- Imports polyfill with cdn external -->
    <script src="https://cdnjs1.com/ajax/libs/webcomponents/webcomponents-lite.min.js"></script>
    <!-- Imports custom element with cdn external -->
    <link rel="import" href="https://cdnjs1.com/ajax/libs/webcomponents/hello-world.html">
</head>
<body>
    <!-- Runs custom element from cdn1, cdn2 -->
    <hello-world who="Unicorn"></hello-world>
</body>
</html>
```

*Is hosting web components in cdnjs possible?*

**Comment by MattIPv4** [comment, marked as answer]
👋 cdnjs does not support `.html` files. You can see the list of extensions we support via our API: https://api.cdnjs.com/whitelist
